modules/devguide/examples/Cloud.java

```diff
@@ -82,40 +80,35 @@ public static void main(String... args) throws Exception {
         trustStore.setCertificateEntry("server", decodeCertificates(Collections.singletonList(CERT)).get(0));
 
         ClusterEnvironment env = ClusterEnvironment.builder()
-            .securityConfig(SecurityConfig.enableTls(true)
-                .trustManagerFactory(InsecureTrustManagerFactory.INSTANCE))
-            .ioConfig(IoConfig.enableDnsSrv(true))
-            .build();
+                .securityConfig(
+                        SecurityConfig.enableTls(true).trustManagerFactory(InsecureTrustManagerFactory.INSTANCE))
+                .ioConfig(IoConfig.enableDnsSrv(true)).build();
```

Comment:
`.build` should be formatted on following line, as it's part of the pipeline, not related to the ioConfig